README.md

```diff
@@ -1,18 +1,27 @@
 # Hello-Microverse
 
-> One paragraph statement about the project.
-> Description the project.
+> One h1 Tittle about the project.
+> Create branches with different base branches.
+> Understand the concept of version control.
+> Follow GitHub flow.
+> Open pull requests in GitHub.
+> Write descriptive, easy to understand commit messages.
+> Write short, detailed, easy to understand descriptions of pull requests.
+> Write a descriptive and easy to understand README in business English for every project submitted.
+> Demonstrate an ability to submit a project for code review..
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I suggest you move these texts in your pull request description to make it professional instead of using thwm (texts) in the README.

Reply:
Hi Tresor!, nice to talk to you.
I already modified all the revisions that you observed me. But I don't know how to request a new review. Could you guide me, please?
Thanks!

Reply:
Is no need to do another pull request... just submitting again with the same link?

Reply:
"Kindly make sure Linters are set up correctly for this repository and the result is green for all checks(Linter should report NO errors)."

where should I see the list report?

Reply:
Hi @MarcoAurelioAntonio 

Just send me a link of the zoom call on slack, and I can show you how to do so. Slack_Username: @Tresor Sawasawa